parser.py

```diff
@@ -109,12 +112,21 @@ def match_subject(tag):
     def course_action(self, course_unique):
         """Return the action for the course unique"""
         tag = self.soup.find("a", id=self.ALL_COURSES, text=course_unique)
+
         if not tag:
             logging.warning(u"Couldn't find the course '{0}'".format(course_unique))
             return None
 
         return tag["id"]
 
+    def disambiguation_action(self):
+        tags = self.soup.find_all("a", id=self.ALL_CAREERS)
+
+        if not tags:
+            return None
+
+        return tags[len(tags)-1]["id"]
```

Comment:
You should be able to just write this as `tags[-1]["id"]` - Is this trying to choose the main campus option? Is it always guaranteed to be the last one?